daemon/graphdriver/counter.go

```diff
@@ -54,6 +54,9 @@ func (c *RefCounter) incdec(path string, infoOp func(minfo *minfo)) int {
 	}
 	infoOp(m)
 	count := m.count
+	if count <= 0 {
```

Comment:
I think this makes more sense in `Decrement`?

Reply:
Thanks for review.

It's doable. But `infoOp` should only handles `minfo *`. I think Function `incdec` seems a general place to take care of cleanup. Unless we want a another refactor of the whole code.

I think this fix is pretty simple. How do you want me to handle that?

Reply:
It's fine, was just a suggestion.